drivers/bcmxcp_ser.c

```diff
@@ -37,7 +37,7 @@ static void send_command(unsigned char *command, size_t command_length)
 {
 	int retry = 0;
 	ssize_t sent;
-	unsigned char sbuf[128];
+	unsigned char sbuf[1024];
```

Comment:
Just in case, where did this particular number come from? (I understand the change originates from Coverity complaints, but did it say what buffer is too small and by how much?)

Reply:
Hot fix to pass coverity test